apm-agent-api/src/main/java/co/elastic/apm/api/TransactionImpl.java

```diff
@@ -77,6 +77,24 @@ public Transaction addLabel(String key, boolean value) {
         return this;
     }
 
+    @Nonnull
+    @Override
+    public Transaction addCustomContext(String key, String value) {
+        return this;
```

Comment:
What about your nice comments referring to the corresponding instrumentation? I like those 😄